### ISSUE TITLE
Can you please add test code as well along with train

### ISSUE THREAD

**Issue by vi3k6i5** [aymericdamien/TensorFlow-Examples#2]
For example in this code TensorFlow-Examples/examples/2 - Basic Classifiers/linear_regression.py

please add how to generate a test set and make prediction for it and validate it as well.

**Comment by aymericdamien** [comment]
This is a linear regression, to make a prediction, just use the formula: y = w.x + b, with x your input and w and b calculated during the learning process. The example data were generated randomly. To validate or test a set of data, you can compare the mean square error of your train set and your test set.

I will add a testset to the example and do a cross-validation using mean square error.
